nodeup/pkg/model/packages.go

```diff
@@ -60,7 +60,12 @@ func (b *PackagesBuilder) Build(c *fi.NodeupModelBuilderContext) error {
 		c.AddTask(&nodetasks.Package{Name: "conntrack-tools"})
 		c.AddTask(&nodetasks.Package{Name: "ebtables"})
 		c.AddTask(&nodetasks.Package{Name: "ethtool"})
-		c.AddTask(&nodetasks.Package{Name: "iptables"})
+		if b.Distribution == distributions.DistributionAmazonLinux2023 {
```

Comment:
I would think this condition is more based on CNI than specific distro. Or doesn't this package exist in the other RHEL-based distros?

Reply:
i have no idea about other RHEL based distro(s), we can certainly revisit once i get this PR working :)